pkgs/by-name/ko/kotatsu-dl/package.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs }:
```

Comment:
Never use `pkgs` in the top-level package inputs. Always directly import all that you need. So `stdenv`, `fetchFromGithub`, ... It allows for easier overrides of
the package.